AnyText/AnyText.Core/Rules/MultiRuleApplication.cs

```diff
@@ -161,6 +163,48 @@ public override object GetValue(ParseContext context)
             }
         }
 
+        public override void GetFoldingRanges(ICollection<FoldingRange> result)
+        {
+            base.GetFoldingRanges(result);
+
+            if (Rule is ZeroOrMoreRule zeroOrMoreRule && zeroOrMoreRule.InnerRule.IsImports())
```

Comment:
Ich hatte gemeint, dass wir für diese Sequenz eine virtuelle Methode in Rule vorsehen, die dann für ZeroOrMoreRule, SequenceRule und ParanthesesRule passend überschrieben wird. Aber dann wird es schwieriger, die Methode GetFoldingRange hier wieder zu verwenden.